_posts/2022-11-03-autoscaling-functions.md

```diff
@@ -0,0 +1,212 @@
+---
+title: "Autoscaling for OpenFaaS Functions"
+description: "Learn how and why to use autoscaling for OpenFaaS Functions"
+date: 2022-11-04
+image: /images/2022-scaling-functions/background.jpg
+categories:
+- autoscaling
+- prometheus
+- functions
+author_staff_member: han
+---
+
+Learn how and why to use autoscaling for OpenFaaS Functions
```

Comment:
We redesigned the autoscaler in OpenFaaS to match the patterns we were seeing for customer workloads. In this article we'll compare and contrast the three new strategies with the original Community Edition approach from 2017.